src/test/kotlin/com/ampnet/headlesscmsservice/controller/MailControllerTest.kt

```diff
@@ -188,9 +188,12 @@ class MailControllerTest : ControllerTestBase() {
     @WithMockCrowdfundUser
     fun mustBeAbleUpdateMail() {
         val request = MailUpdateRequest("Invitation New Title", invitationMailContent)
-        verify("Controller will return default email") {
+        verify("Controller will return updated mail") {
             val result = mockMvc.perform(
-                post("/mail/$COOP/${MailType.INVITATION_MAIL}/${Lang.EN}")
+                post(
+                    "/mail/$COOP/${MailType.INVITATION_MAIL.toString().toLowerCase()}" +
+                        "/${Lang.EN.toString().toLowerCase()}"
```

Comment:
There is no need for `.toLowerCase()` see file: `Lang.kt` -> `fun Lang.name() = this.name.toLowerCase()`
You can do this for `MailType` to space all `.toLowerCase()` calls.